README.md

```diff
@@ -13,16 +13,18 @@ var cleankill = require('cleankill');
 To register a handler:
 
 ```js
-cleankill.onInterrupt(function(done) {
-  // do things.
-  done();
+cleankill.onInterrupt(() => {
```

Comment:
nit: if you use an arrow function that just returns an expression, don't use statements...
```javascript
  cleankill.onInterrupt(() => new Promise((resolve) => {
    // do things.
    resolve();
  }));
```